src/scripts/calculateScrollAlpha.ts

```diff
@@ -0,0 +1,6 @@
+export default function calculateScrollAlpha():string {
```

Comment:
I think you could probably have this return `: number` since `.toFixed()` returns a float, which is a number in TS. It probably will still work as a string, but using the numeric value is probably easier and more precise. Eg `0.00` instead of `'0.00'`

Reply:
I initially thought that it was float too (given two decimal places), but checking the [docs for toFixed()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toFixed#return_value) I saw its return type is string. 

I think I'd then need to parse it to a float, so instead of adding that additional code, and since I'm not doing any further calculations on the alpha value, I kept my code as having alpha as a string - which seems to work in my application of the alpha value. 

Do you think it would be beneficial to change `alpha` to a float instead of string?

Reply:
Floats are sort of halfway between a string and an int, so generally they can go either way. You're right about `.toFixed()` though, my bad, it does seem to return a true string. 

The biggest difference is precision, and how easy they are to work with for calculations. Consider the following examples:
```js
10.00 === 10 // true
10.00 === 10.00 // true
10.00 + 12.00 // 22
'10.00' + '12.00' // '10.0012.00'
'10.00' === 10 // false
'10.00' === '10.0' // false
'10.00' === '10.00' // true
```
So as a string, there's a difference between `'10.0'` and `'10.00'` which is useful if you want to display the decimal points (eg. when displaying money), however as a float/number, `10`, `10.0` and `10.000000` are all equivalent, which is useful if you're doing math or calculations. 

All that to say, it's totally your call depending on the application, but unless you're displaying the value, I generally stick with the more numeric option.